From 389d9c8f614bef7badd5e6830c18d4dbdcdc2d0a Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal@vasilek.cz>
Date: Tue, 19 Jul 2022 14:33:30 +0200
Subject: [PATCH] tools: disable kwboot

kwboot only works on omnia and requires libtinfo on host
---
 tools/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/Makefile b/tools/Makefile
index 9f2339666a..dd7ec2ad69 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -222,7 +222,7 @@ ubsha1-objs := os_support.o ubsha1.o lib/sha1.o
 HOSTCFLAGS_ubsha1.o := -pedantic
 
 hostprogs-$(CONFIG_ARCH_KIRKWOOD) += kwboot
-hostprogs-$(CONFIG_ARCH_MVEBU) += kwboot
+hostprogs-$(CONFIG_ARCH_MVEBU) +=
 hostprogs-y += proftool
 hostprogs-$(CONFIG_STATIC_RELA) += relocate-rela
 hostprogs-$(CONFIG_RISCV) += prelink-riscv
-- 
2.37.1

